editor/editor_node.h

```diff
@@ -142,22 +142,10 @@ class EditorNode : public Node {
 		EDIT_REVERT,
 		TOOLS_ORPHAN_RESOURCES,
 		TOOLS_CUSTOM,
-		RESOURCE_NEW,
-		RESOURCE_LOAD,
 		RESOURCE_SAVE,
```

Comment:
`RESOURCE_SAVE` and `RESOURCE_SAVE_AS` can't be moved together with the rest?

Reply:
im not sure, there is more resource saving logic in EditorNode, that wasn't needed to complete the inspector dock rework, i might need to dig deeper into that to see where those functions are needed.